src/qjs_buffer.c

```diff
@@ -103,14 +158,104 @@ static const JSCFunctionListEntry qjs_buffer_export[] = {
 
 
 static const JSCFunctionListEntry qjs_buffer_props[] = {
+    JS_CFUNC_MAGIC_DEF("alloc", 3, qjs_bufferobj_alloc, 0),
+    JS_CFUNC_MAGIC_DEF("allocUnsafe", 3, qjs_bufferobj_alloc, 1),
+    JS_CFUNC_DEF("byteLength", 2, qjs_buffer_byte_length),
+    JS_CFUNC_DEF("compare", 6, qjs_buffer_compare),
+    JS_CFUNC_DEF("concat", 1, qjs_buffer_concat),
     JS_CFUNC_DEF("from", 3, qjs_buffer_from),
     JS_CFUNC_DEF("isBuffer", 1, qjs_buffer_is_buffer),
+    JS_CFUNC_DEF("isEncoding", 1, qjs_buffer_is_encoding),
 };
 
 
 static const JSCFunctionListEntry qjs_buffer_proto[] = {
+    JS_CFUNC_DEF("compare", 5, qjs_buffer_prototype_compare),
+    JS_CGETSET_DEF("constructor", qjs_buffer_prototype_constructor, NULL),
```

Comment:
constructor of Buffer proto  need to be usual "configurable, writable, not enumerable" property,
which is initially set to Buffer.

Example of issue:

  var b = Buffer.from('a');
  var buffer_old = Buffer;
  Buffer = 1;

as result got wrong::

  b.constructor === 1;

while should produce:

   b.constructor === buffer_old;;

Reply:
Otherwise all parts of the PR looks good to me.